docs/topics_5/web-install-zip.adoc

```diff
@@ -12,17 +12,10 @@ The {WebName} has been tested with Chrome and Firefox.
 * OpenJDK 1.8, OpenJDK 11, Oracle JDK 1.8, or Oracle JDK 11
 * 16 GB RAM
 
-macOS has the following additional prerequisites:
+macOS has the following additional prerequisite:
 
 * Set the `maxproc` value to `2048` or greater.
 
-ifeval::["{ProductVersion}" == "5.0.0"]
```

Comment:
There are 3 other modules that need to be updated because they also have maxfiles

Reply:
Done.

Reply:
BTW, it's insane to update 4 different files. See if you can modularize this